.gitlab-ci.yml

```
@@ -111,8 +111,14 @@ check-line-width:
   interruptible:                   true
   allow_failure:                   true
 
-
-
+alert-pending-release:
+  stage:                           test
+  image:                           parity/tools:latest
+  only:
+    - tags
+    - /^v[0-9]+\.[0-9]+\.[0-9].*$/        # i.e. v1.0.1, v2.1.0rc1
```

```suggestion
    - /^v[0-9]+\.[0-9]+\.[0-9]+.*$/        # i.e. v1.0.1, v2.1.0rc1
```

I guess `v0.1.11` is a pretty valid tag =)

---

Hmm, actually the original regexp would of course work — it just feels more natural with all digits being captured logically